lib/ddtrace/monkey.rb

```diff
@@ -16,8 +16,10 @@ module Monkey
     #   can be call twice but should just do nothing the second time.
     # - patched?, which returns true if the module has been succesfully
     #   patched (patching might have failed if requirements were not here)
-    @patchers = { elasticsearch: Datadog::Contrib::Elasticsearch::Patcher,
-                  redis: Datadog::Contrib::Redis::Patcher }
+    @patchers = {
+      elasticsearch: Datadog::Contrib::Elasticsearch::Patcher,
+      redis: Datadog::Contrib::Redis::Patcher
```

Comment:
Any chance we could have Sinatra here to ? Does it make sense ? Might be simpler now that it's not enabled by default. I think being able to call `Datadog::Monkey.get_patched_modules` and now wether Sinatra is instrumented or not could be useful when trying to find out "what's going on". But I let you decide, for Rails for instance we did not do it in the first place because it's a framework and, initially, patching was meant to be "all automagic".